### PR TITLE
Fixes #14675: Start tomcat on upgrade before Foreman migrations

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: hooks/pre/30-upgrade.rb [hunk 1]
```diff
@@ -19,6 +19,10 @@ def migrate_candlepin
   Kafo::Helpers.execute("/usr/share/candlepin/cpdb --update --password #{Kafo::Helpers.read_cache_data('candlepin_db_password')}")
 end
 
+def start_tomcat
+  Kafo::Helpers.execute('katello-service start --only tomcat')
```
Comment (stbenjam) [inline hooks/pre/30-upgrade.rb]: On el6, the service is called tomcat6 :-(

Comment (stbenjam) [inline hooks/pre/30-upgrade.rb]: But the script ignores non-existent services, so you can say `--only tomcat,tomcat6`

**Comment by ehelms** [comment]
@stbenjam Updated based on your suggestion

**Comment by stbenjam** [comment]
ACK

**Pull request merged**